src/server/routes/questionnaire.route.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { createQuestionnaireController } from '../../controller/createQuestionnaireController';
+import { getHealthController } from '../../controller/healthController';
+import { getDreamControllerById } from '../../controller/getDreamControllerById';
+const router = express.Router();
+
+router.post('/create',  createQuestionnaireController);
```

Comment:
When create a resource in database, there is no need to add the verb `create`. It is an standard. So you can define `router.post('/', ...);`

Reply:
Updated in https://github.com/Sly-Agustin/nosql-practice/commit/32e06334cb14ef863c9d06941c53a4b8bee84c7b